src/main/Main.java

```diff
@@ -1,9 +1,94 @@
+import java.lang.Math;
 
 public class Main {
 	
+	// CONSTANT
```

Comment:
We should place them in a separate class file only containing the constants. So it's easy to import them to the classes that need them.

Reply:
There is only one constant and that is PI. The rest are variable declarations. You want to move all of them to a separate file?